py_gtfs_rt_ingestion/tests/test_configuration.py

```diff
@@ -31,43 +30,3 @@ def test_filname_parsing() -> None:
 
     with pytest.raises(ConfigTypeFromFilenameException):
         ConfigType.from_filename("this.is.a.bad.filename.json.gz")
-
-
-def test_get_schema() -> None:
```

Comment:
Configurations don't exist anymore, so i got rid of this.

I could move some things around so we have a suite of tests for GTFS and a suite for GTFSRT as well that would bring these back.